<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateRoutes</class>
 <widget class="QWidget" name="updateRoutes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>570</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Маршрут</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/new/img/icon.jpg</normaloff>:/new/img/icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Точка прибытия</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDistance">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>329</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
    border-width: 1px;
    border-style: solid;
    border-color: black;
	border-radius: 7px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Точка отправдения</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEndLoc">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>329</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
    border-width: 1px;
    border-style: solid;
    border-color: black;
	border-radius: 7px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Расстояние</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditStartLoc">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>133</y>
     <width>329</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
    border-width: 1px;
    border-style: solid;
    border-color: black;
	border-radius: 7px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
	background-color: #0089D9;
	border: none;
	color: white;
	border-radius: 7px;
}

QPushButton:pressed{
	background: #00598C;}</string>
   </property>
   <property name="text">
    <string>Изменить маршрут</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
